src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -2316,3 +2316,10 @@ OperatorImplementsEquality(Oid opno)
 
 	return equalityOperator;
 }
+
+
+bool
+QueryContainsDistributedTableRTE(Query *query)
```

Comment:
needs a function comment

Reply:
Using FindNodeMatchingCheckFunction(). We don't this function anymore. Removed it.